src/Rings/AbelianClosure.jl

```diff
@@ -928,6 +928,41 @@ AbstractAlgebra.promote_rule(::Type{QQAbFieldElem}, ::Type{ZZRingElem}) = QQAbFi
 
 AbstractAlgebra.promote_rule(::Type{QQAbFieldElem}, ::Type{QQFieldElem}) = QQAbFieldElem
 
+###############################################################################
+#
+#  isinteger, is_rational, is_integral
+#
+###############################################################################
+
+@doc raw"""
+    isinteger(a::QQAbFieldElem)
+
+Return whether $a$ is an integer.
+"""
```

Comment:
Should we really add this kind of docstring? Isn't this already covered by the Julia's primary docstring for `isinteger`? In fact looking at the output of `?isinteger` in the REPL I think we could basically remove all of them.

Reply:
Fair enough. I am probably as well. And some of them are not trivial to remove because e.g. the Nemo doc includes several of them (but I think in all cases this could be avoided by inserting some general text like "For this ring the following things are implemented: `+`, `-`, `isinteger`, " 

Anyway, no reason to hold up this PR I think.